mcs/class/corlib/corefx/RuntimeFeature.cs

```diff
@@ -0,0 +1,14 @@
+namespace System.Runtime.CompilerServices
+{
+	partial class RuntimeFeature
+	{
+		// https://github.com/dotnet/coreclr/blob/397aaccb5104844998c3bcf6e9245cc81127e1e2/src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeFeature.CoreCLR.cs#L9-L10
+#if WASM || (FULL_AOT_RUNTIME && !MONOTOUCH)
+		public const bool IsDynamicCodeSupported = false;
+		public const bool IsDynamicCodeCompiled = false;
```

Comment:
I'm a bit confused by the intention of this PR.
* does `WASM` imply `FULL_AOT_RUNTIME`?
* why is `FULL_AOT_RUNTIME && ...` needed? every usage of those two constants are guarded by `!FULL_AOT_RUNTIME`.

WebAssembly _can_ be shipped with the interpreter, so why would dynamic code execution be disabled?
* note that WASM _can_ have an interpreter available, so dynamic code execution

Reply:
That's right, Martin we discussed that the fields can only be overwritten by linker for wasm/mobile

Reply:
The conditional in it's current form is supposed to mean `WASM` and all the `testing_aot_*` profiles.  I added the `MONOTOUCH` because without it the API Diff broke.

Reply:
Note that the constants are strictly required - you cannot use properties and the linker - at least not without additional linker work.

Reply:
See my comment below; I took out those changes, so we can move along with this one.